declarations/Booking.com.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Booking.com",
+  "documents": {
+    "Terms of Service": {
```

Indeed, as @martinratinaud highlights, the referenced document is _not_ Booking.com’s rental service conditions for their `end user` (people renting properties), but for their `commercial partners, content creators in a platform` (property owners). I thus suggest:
```suggestion
    "Commercial Terms": {
```